superset/assets/src/visualizations/deckgl/CategoricalDeckGLContainer.jsx

```diff
@@ -48,19 +48,20 @@ export default class CategoricalDeckGLContainer extends React.PureComponent {
    */
 
   /* eslint-disable-next-line react/sort-comp */
-  static getDerivedStateFromProps(nextProps) {
-    const fd = nextProps.slice.formData;
+  static getDerivedStateFromProps(nextProps, currentState) {
+    const fd = nextProps.payload.form_data;
```

Comment:
What is the difference between `slice.formData` and `payload.form_data`?

Reply:
My understanding is `slice.formData` is also updated when the controls change in the explore view. `slice` here is the container component (`Chart.jsx`)